docs/sources/variables/chained-variables.md

```diff
@@ -0,0 +1,174 @@
+
+#### app variable
+
+The query for this variable basically says, "Give me all the applications that exist."
+
+```
+apps.*
+```
+
+The values returned are `backend`, `country`, `fakesite`, and `All`.
+
+#### server variable
+
+The query for this variable basically says, "Give me all servers for the currently chosen application." 
+
+```
+apps.$app.*
+```
+
+By itself, this query is nonsense, because `$app` has no value. However, if the user selects `backend`, then the query changes to:
+
+```
+apps.backend.*
+```
+
+The query returns all servers associated with `backend`, including `backend_01`, `backend_02`, and so on.
+
+If the user selects `fakesite`, then the query changes to:
+
+```
+apps.fakesite.*
+```
+
+The query returns all servers associated with `fakesite`, including `web_server_01`, `web_server_02`, and so on.
+
+#### More variables
+
+> **Note:** This example is theoretical. The Graphite server used in the example does not contain CPU metrics.
+
+The dashboard stops at two levels, but you could keep going. For example, if you wanted to get CPU metrics for selected servers, you could copy the `server` variable and extend the query so that it reads:
+
+```
+apps.$app.$server.cpu.*
+```
+
+This query basically says, "Show me the CPU metrics for the selected server."
+
+Depending on what variable options the user selects, you could get queries like:
+
+```
+apps.backend.backend_01.cpu.*
+apps.backend.backend_02+backend_03.cpu.*
+apps.fakesite.web_server_01.cpu.*
+```
+
+### InfluxDB example
+
+In this example, you have several data centers. Each data center has a different subset of hosts. It is based on the [InfluxDB Templated](https://play.grafana.org/d/000000002/influxdb-templated?orgId=1).
+
+In this example, when the user changes the value of the `datacenter` variable, it changes the dropdown options returned by the `host` variable. The `host` variable uses the **Multi-value** option and **Include all option**, allowing users to select some or all options presented at any time. The `datacenter` does not use either option, so you can only select one data center at a time.
+
+#### datacenter variable
+
+The query for this variable basically says, "Give me all the data centers that exist."
+
+```
+SHOW TAG VALUES  WITH KEY = "datacenter" 
+```
+
+The values returned are `America`, `Africa`, `Asia`, and `Europe`.
+
+#### host variable
+
+The query for this variable basically says, "Give me all hosts for the currently chosen data center." 
+
+```
+SHOW TAG VALUES WITH KEY = "hostname" WHERE "datacenter" =~ /^$datacenter$/
+```
+
+By itself, this query is nonsense, because `$datacenter$` has no value. However, if the user selects `America`, then the query changes to:
```

Comment:
Don't really understand the nonsense part. Why doesn't `$datacenter` has a value. Since multi value is not selected the first value returned from data center variable will be selected.

```suggestion
By itself, this query is nonsense, because `$datacenter` has no value. However, if the user selects `America`, then the query changes to:
```